UI/window-basic-main-menus.cpp

```diff
@@ -0,0 +1,234 @@
+#include "window-basic-main.hpp"
+#include <sstream>
+
+void OBSBasic::SetDeinterlacingMode()
+{
+	QAction *action = reinterpret_cast<QAction *>(sender());
```

Comment:
`sender()` is a last resort solution provided by Qt which does not respect POO programming.

I don't think it's a good idea to heavily rely on it.

Reply:
This PR just copies the code from OBSBasic, so nothing has changed. We can always change the sender() code in a future PR.